Release v1.3.5 does not install without yarn

Hi,

The 1.3.5 release on npm won't install on a standard node installation. The `postinstall` script tries to do

```
cd ./integration && yarn
```
which results in:
```
The system cannot find the path specified.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tslint-lines-between-class-members@1.3.5 postinstall: `cd ./integration && yarn`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tslint-lines-between-class-members@1.3.5 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
````
Surely doing anything with yarn should not be required on postinstall?

even if yarn is installed on the system, same error:
```
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! tslint-lines-between-class-members@1.3.5 postinstall: `cd ./integration && yarn`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the tslint-lines-between-class-members@1.3.5 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/valeri/.npm/_logs/2019-07-20T21_51_56_905Z-debug.log
valeri@foo (foo*) ~/workspace/backup: yarn --version
1.17.3
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -37,6 +37,7 @@ yarn compile:watch
 And then in another terminal run the linting script from inside the sample project:
 ```bash
 cd ./integration/
+yarn
 
 yarn lint
 # or

--- a/package.json
+++ b/package.json
@@ -3,7 +3,6 @@
   "version": "1.3.5",
   "description": "Custom rule for TSLint to enforce blank lines between class methods - achieves a similar thing to lines-between-class-members in ESLint",
   "scripts": {
-    "postinstall": "cd ./integration && yarn",
     "lint": "tslint --config tslint.json --exclude node_modules **/*.ts",
     "test": "ava-ts src/**/test/**/*.spec.ts --verbose",
     "test:watch": "concurrently \"yarn test --watch\" \"yarn compile:watch\" -n \"ava,tsc\" -p -c \"magenta,blue\"",

